### ISSUE TITLE
Crash on Startup (Server)

### ISSUE THREAD

**Issue by ghost** [MrMelon54/illegal-stack-fixer-fabric-1.16#1]
MC-Version: 1.16.5

```
[16:01:58] [main/FATAL]: Mixin apply failed IllegalStackFixer.mixins.json:MixinChestBlock -> net.minecraft.class_2281: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow method method_24167 in IllegalStackFixer.mixins.json:MixinChestBlock was not located in the target class net.minecraft.class_2281. Using refmap IllegalStackFixer-refmap.json
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow method method_24167 in IllegalStackFixer.mixins.json:MixinChestBlock was not located in the target class net.minecraft.class_2281. Using refmap IllegalStackFixer-refmap.json
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachSpecialMethod(MixinPreProcessorStandard.java:423) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachShadowMethod(MixinPreProcessorStandard.java:399) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachMethods(MixinPreProcessorStandard.java:327) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:286) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:264) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1272) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:287) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) ~[fabric-server-launch.jar:?]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) ~[fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157) ~[fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150) ~[fabric-server-launch.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_261]
	at net.minecraft.class_3523.<clinit>(class_3523.java:13) ~[intermediary-minecraft_server.1.16.5.jar:?]
	at net.minecraft.class_5471.<clinit>(class_5471.java:11) ~[intermediary-minecraft_server.1.16.5.jar:?]
	at net.minecraft.class_5458.method_30573(class_5458.java:44) ~[intermediary-minecraft_server.1.16.5.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:75) ~[intermediary-minecraft_server.1.16.5.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:684) [?:1.8.0_261]
	at net.minecraft.class_5458.<clinit>(class_5458.java:74) [intermediary-minecraft_server.1.16.5.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:237) [intermediary-minecraft_server.1.16.5.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:42) [intermediary-minecraft_server.1.16.5.jar:?]
	at net.minecraft.server.Main.main(Main.java:87) [intermediary-minecraft_server.1.16.5.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_261]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_261]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_261]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_261]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:28) [fabric-server-launch.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_261]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_261]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_261]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_261]
	at net.fabricmc.loader.launch.server.FabricServerLauncher.launch(FabricServerLauncher.java:62) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.server.FabricServerLauncher.setup(FabricServerLauncher.java:106) [fabric-server-launch.jar:?]
	at net.fabricmc.loader.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:50) [fabric-server-launch.jar:?]

```

**Comment by mrmelon54** [comment]
This didn't happen during my testing (obviously). I'll have a look into it.

**Comment by mrmelon54** [comment]
Fix coming in v1.0.1
